src/cdk/constructs/domain-manager/resource/route53-manager.ts

```diff
@@ -1,103 +1,110 @@
-import { CfnOutput, Construct, Duration } from '@aws-cdk/core'
-import { Topic } from '@aws-cdk/aws-sns'
-import { SingletonFunction, Runtime, AssetCode } from '@aws-cdk/aws-lambda'
-import { FollowMode } from '@aws-cdk/assets'
-import { SnsEventSource } from '@aws-cdk/aws-lambda-event-sources'
 import { AccountPrincipal, CompositePrincipal, Effect, ManagedPolicy, PolicyStatement, Role, ServicePrincipal } from '@aws-cdk/aws-iam'
-import { MiraConfig } from '../../../../config/mira-config'
+import { Code, Runtime, SingletonFunction } from '@aws-cdk/aws-lambda'
+import { SnsEventSource } from '@aws-cdk/aws-lambda-event-sources'
+import { Topic } from '@aws-cdk/aws-sns'
+import { CfnOutput, Construct, Duration } from '@aws-cdk/core'
 import path from 'path'
+import { MiraConfig } from '../../../..'
 import { MiraStack } from '../../../stack'
+import { ROUTE53_MANAGER_POLICY_BOUNDARY } from './route53-manager-access-role'
+
+export const DOMAIN_SUBSCRIPTION_TOPIC = 'DomainSubscriptionTopic'
+export const CROSS_ACCOUNT_DOMAIN_MANAGER_ROLE = 'CrossAccountDomainManagerRole'
+
 export class Route53Manager extends MiraStack {
   constructor (parent: Construct) {
-    const id = MiraConfig.getBaseStackName('Route53Manager')
-    super(parent, id)
-    const account = MiraConfig.getEnvironment()
+    super(parent, 'Route53Manager')
+
     const { hostedZoneId } = MiraConfig.getDomainConfig()
+
     if (!hostedZoneId) {
```

Comment:
Wishing we had one place in our code for this where we could parse the configs, and perhaps offer an API where we perform checks.  I think the `MiraConfig` may currently be the best place, but I think adding in items for the domain configuration may create confusion.